src/drawer.js

```diff
@@ -300,12 +300,13 @@ $.Drawer.prototype = {
      * @param {Float} [scale=1] - Apply a scale to tile position and size. Defaults to 1.
      * @param {OpenSeadragon.Point} [translate] A translation vector to offset tile position
      */
-    drawTile: function(tile, drawingHandler, useSketch, scale, translate) {
+    drawTile: function(tile, drawingHandler, useSketch, scale, translate, imageSmoothingEnabled) {
         $.console.assert(tile, '[Drawer.drawTile] tile is required');
         $.console.assert(drawingHandler, '[Drawer.drawTile] drawingHandler is required');
 
         if (this.useCanvas) {
             var context = this._getContext(useSketch);
+            context.imageSmoothingEnabled = imageSmoothingEnabled;
```

Comment:
To maintain backwards compatibility, if `imageSmoothingEnabled` is omitted it should default to true, like so:

`context.imageSmoothingEnabled = imageSmoothingEnabled === undefined ? true : imageSmoothingEnabled;`

Same in `blendSketch`.